src/acquire.c

```diff
@@ -317,13 +317,14 @@ acquire_configure(struct AcquireRuntime* self_,
                 TRACE("Configured video stream %d.", istream);
             } else {
                 TRACE("Failed to configure video stream %d.", istream);
+                self->valid_video_streams &= ~(1 << istream);
             }
         }
     }
     TRACE("Valid video streams: code %#04x", self->valid_video_streams);
-    self->state = self->valid_video_streams > 0
-                    ? DeviceState_Armed
-                    : DeviceState_AwaitingConfiguration;
```

Comment:
Don't you want to set state to `DeviceState_Armed`?

Reply:
Per discussion, could be running. Still needs to go to Armed if it was AwaitingConfiguration.